packages/@uppy/xhr-upload/src/index.ts

```diff
@@ -106,37 +101,12 @@ const defaultOptions = {
   fieldName: 'file',
   method: 'post',
   allowedMetaFields: true,
-  responseUrlFieldName: 'url',
   bundle: false,
   headers: {},
   timeout: 30 * 1000,
   limit: 5,
   withCredentials: false,
   responseType: '',
-  getResponseData(responseText) {
-    let parsedResponse = {}
-    try {
-      parsedResponse = JSON.parse(responseText)
-    } catch {
-      // ignore
-    }
-    // We don't have access to the B (Body) generic here
-    // so we have to cast it to any. The user facing types
-    // remain correct, this is only to please the merging of default options.
-    return parsedResponse as any
-  },
-  getResponseError(_, response) {
-    let error = new Error('Upload error')
-
-    if (isNetworkError(response)) {
```

Comment:
don't we still need this code?

Reply:
The option `getResponseError` no longer exists.

Reply:
sure, but don't we still need to convert `isNetworkError(response)` to a `new NetworkError(error, response)`? I don't see that code anymore

Reply:
I don't think so. We build the error here

https://github.com/transloadit/uppy/blob/33033fccc7f256dd22623301071bba1c74736f9d/packages/%40uppy/xhr-upload/src/index.ts#L222-L238

Reply:
ah. so it was dead/redundant code.